.github/workflows/deploy.yml

```diff
@@ -72,7 +76,7 @@ jobs:
           path: gh-pages
 
       - name: sync static files
-        if: ${{ github.event.inputs.set_current }}
+        if: ${{ github.event.inputs.set_stable }}
         run: |
```

Comment:
I don't understand why this only happens for `.set_current == true`. Does this need to happen for `.set_latest` too?

Reply:
I think we used to have other stuff in the `doc` folder including the "suite design guide" and other things that were version-pinned against Cylc which we wouldn't necessarily want to update for a pre-release.

Now the only thing we have there is the `joss-paper` so I guess we could update with every run, just need to make sure that doesn't mess up the version/format links.